### ISSUE TITLE
Removal of `/usr/share/man/*` from Debian based images prevents customization

### ISSUE THREAD

**Issue by dhs-rec** [jenkinsci/docker-agent#527]
### Jenkins and plugins versions report

[As mentioned in #515]
While creating a customized agent image based on the `bookworm` one, we ran into the following error while trying to install `ca-certificates-java` (which depends on `default-jre`):
```
#9 8.588 update-alternatives: using /usr/lib/jvm/java-17-openjdk-amd64/bin/java to provide /usr/bin/java (java) in auto mode
#9 8.589 update-alternatives: error: error creating symbolic link '/usr/share/man/man1/java.1.gz.dpkg-tmp': No such file or directory
#9 8.589 dpkg: error processing package openjdk-17-jre-headless:amd64 (--configure):
#9 8.589  installed openjdk-17-jre-headless:amd64 package post-installation script subprocess returned error exit status 2
```
`update-alterntatives` tries to create a symlink to a newly installed man-page and fails because `/usr/share/man/man1` has been removed from the image.

### What Operating System are you using (both controller, and any agents involved in the problem)?

Debian based `jenkins/agent` Docker images.

### Reproduction steps

Add customization on top of the provided images. Adding a package that contains a man page will fail because of `/usr/share/man/*` (and probably also `/usr/share/doc`) have been removed from the image (`Debian/Dockerfile`, line 82).

### Expected Results

New packages should be installable w/o problem.

### Actual Results

Adding a package with man-pages fails, see above.

**Comment by dduportal** [comment]
Hi @dhs-rec thanks for the issue. Would you be willing to send a pull request to contribute this fix?

**Comment by dduportal** [comment]
Fixed by #528  and released in https://github.com/jenkinsci/docker-agent/releases/tag/3160.vd76b_9ddd10cc-3.
